demos/2a03/prg/mmc3.cfg

```diff
@@ -77,14 +77,15 @@ SEGMENTS {
    BSS:        load = RAM,                 type = bss, align = $100, define = yes;
    RAM:        load = RAM,                 type = bss, start = $0300;
    HEADER:     load = HDR,                 type = ro,  align = $10;
+# Combine Music 6 and Music 7 into the same PRG bank to test variable address loading
    MUSIC_0:    load = PRG0,                type = ro;
    MUSIC_1:    load = PRG1,                type = ro;
    MUSIC_2:    load = PRG2,                type = ro;
    MUSIC_3:    load = PRG3,                type = ro;
    MUSIC_4:    load = PRG4,                type = ro;
    MUSIC_5:    load = PRG5,                type = ro;
    MUSIC_6:    load = PRG6,                type = ro;
-   MUSIC_7:    load = PRG7,                type = ro;
+   MUSIC_7:    load = PRG6,                type = ro;
```

Comment:
For testing the address changes, i moved MUSIC_7 into prg 6 so it starts at a different address. I chose this one because the other songs are too big to share banks :P